docs/empower.rst

```diff
@@ -0,0 +1,47 @@
+Empower
+=======
+
+********
+Overview
+********
+
+The Empower class allows you to interact with the Empower API. Documentation for the Empower API can be found
+in their `GitHub <https://github.com/getempower/api-documentation/blob/master/README.md>`_ repo.
+
+.. note::
+    The Empower API only has a single endpoint to access all account data. As such, it has a very high overhead. This
+    connector employs caching in order to allow the user to specify the tables to extract without additional API calls.
```

Comment:
More of a personal question, coming from lack of experience, but wondering what caching is?

Reply:
So, there is only one endpoint and it grabs a giant JSON blob that is a bit of a pain to parse through and convert to a tabular-ish format. I've broken up that JSON into multiple functions so that folks can grab the bits of data that they want.

However, that means that every single time you call it function, it is grabbing a lot of extraneous data. So, the caching just stores the blob and extracts from it rather than calling the Empower server again.

At the time, it seemed like a cute idea, but perhaps its unnecessary.